api/src/main/java/org/openmrs/module/appframework/service/AppFrameworkServiceImpl.java

```diff
@@ -339,20 +336,23 @@ public boolean checkRequireExpression(Requireable candidate, AppContextModel con
 				// all Map-type properties to JSON and use this to define objects directly within the ScriptEngine.
 				// (Properties that are not Maps don't need this special treatment.)
 				Bindings bindings = new SimpleBindings();
-				Map<String, Object> mapProperties = new HashMap<String, Object>();
+				ObjectMapper objectMapper = new ObjectMapper();
+				Map<String, Object> mapProperties = new HashMap<>();
```

Comment:
Is there any reason why you changed the constructor for mapProperties?

Reply:
Oh yah!!! Talk of the diamond operator and the like. Thanks for reminding the old man about this. 😊